src/Microsoft.Framework.DependencyInjection.Abstractions/project.json

```diff
@@ -8,7 +8,7 @@
         "net45": {
         },
         "dnx451": { },
```

Comment:
@ddobric, Oren wrote a nice blog post on the subject: [Demystifying PCLs, .NET Core, DNX and UWP](http://oren.codes/2015/06/16/demystifying-pcls-net-core-dnx-and-uwp-redux/)